docs/eBpfExtensions.md

```diff
@@ -73,7 +73,7 @@ functions that override the global helper functions provided by the eBPF runtime
 structure from provided data and context buffers.
 * `context_destroy`: Pointer to `ebpf_program_context_destroy_t` function that destroys a program type specific
 context structure and populates the returned data and context buffers.
-* `required_irql`: IRQL that the eBPF program runs at.
```

Comment:
Now that there's no minimum irql, does that mean that all eBPF programs might be invoked at PASSIVE level and hence are pre-emptible?  That doesn't seem right to me.

Reply:
I had considered having a minimum / maximum IRQL level for the program information, but wasn't sure if that made sense. Thoughts?

Reply:
Before, `minimum_irql==maximum_irql` was implied by the name `required_irql`.  If they can be different, then yes we need two fields.  If they cannot be different, then `required_irql` is the correct name.

Reply:
Made min and max IRQL explicit parameters.